packages/civic-sandbox/src/state/sandbox/index.js

```diff
@@ -135,9 +137,17 @@ const reducer = (state = INITIAL_STATE, action) => {
       const [findDefaultLayers] = state.sandbox.packages.filter(
         d => d.displayName === action.selectedPackage.displayName
       );
+
+      if (state.selectedPackage === action.selectedPackage.displayName) {
+        return state;
+      }
+
       return {
         ...state,
         selectedPackage: action.selectedPackage.displayName,
+        selectedPackageDescription: action.selectedPackage.description
+          ? action.selectedPackage.description
+          : "A brief description of the selected data collection",
```

Comment:
You can change the line above to this and get the package descriptions to show up:
`selectedPackageDescription: findDefaultLayers.description || "A brief description of the selected data collection"`